tests/test_requests.py

```diff
@@ -1740,6 +1740,50 @@ def test_chunked_upload_does_not_set_content_length_header(self, httpbin):
         assert 'Transfer-Encoding' in prepared_request.headers
         assert 'Content-Length' not in prepared_request.headers
 
+    def test_custom_redirect_mixin(self, httpbin):
+        """This tests using a custom mixin to overwrite `get_redirect_target`.
```

Comment:
Yeah, I noticed that -- but the last few tests added to `TestRequests` had docs, and this one can take a minute of staring at to figure out.  I figured it would be best to submit docs and edit as needed.